apps/examples/dat/Makefile

```diff
@@ -0,0 +1,120 @@
+############################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+# Author: Nitish Ambastha <nitish.a@samsung.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+
+-include $(TOPDIR)/.config
+-include $(TOPDIR)/Make.defs
+include $(APPDIR)/Make.defs
+
+# Power Daemon built-in application info
```

Comment:
Can we Please change the comment "Power Daemon built-in application info" accordingly ?